.dockerignore

```diff
@@ -0,0 +1,21 @@
+Dockerfile
```

Comment:
I always like to have these kind of Dockerfile in the container for reference of people that only see the container.